datafusion/proto/src/logical_plan/to_proto.rs

```diff
@@ -500,7 +246,7 @@ pub fn serialize_expr(
             }
         }
         Expr::Literal(value) => {
-            let pb_value: protobuf::ScalarValue = value.try_into()?;
+            let pb_value: protobuf_common::ScalarValue = value.try_into()?;
```

Comment:
Can we re-export `protobuf_common` from `protobuf` so that import usages don't need to change?